Modify core class references in modContextSettings

JoshuaLuckers opened the pull request:
### What does it do?
Modify core class references in modContextSettings

### Why is it needed?
This solves the error "Class 'modManagerRequest' not found" after upgrading to MODX 3

### Related issue(s)/PR(s)
Resolves issue #14822

JoshuaLuckers committed:
Modify core class references in modContextSettings

This solves the error "Class 'modManagerRequest' not found" after upgrading to MODX 3

Signed-off-by: Joshua Lückers <joshualuckers@me.com>

Mark-H commented on setup/includes/upgrades/common/3.0.0-update-legacy-class-references.php:
```diff
@@ -78,3 +80,9 @@
+$class = modContextSetting::class;
+$table = $modx->getTableName($class);
+
+$modx->updateCollection($class, ['value' => modManagerRequest::class], ['key' => 'modRequest.class']);
```
Should this, like the other examples above, only update where the _value_ is `modManagerRequest` rather than always change the `modRequest.class` setting?

JoshuaLuckers replied:
Yes, you are right!

JoshuaLuckers committed:
Update the value for `modRequest.class` only when needed

Only update where the value is `modManagerRequest` rather than always change the modRequest.class setting.
